runtime/client_http_request.go

```diff
@@ -150,6 +150,7 @@ func (req *ClientHTTPRequest) WriteJSON(
 	httpReq.Header.Set("Content-Type", "application/json")
 
 	req.httpReq = httpReq
+	req.ctx = WithScopeFields(req.ctx, zap.String(scopeFieldRequestMethod, method))
```

Comment:
You shouldn't use `zap.String`; the model of `tally.Scope` is `map[string]string`.